### PR TITLE
Update TemplateTransclusion.yaml

### PR DESCRIPTION
**Pull request by clarakosi**

### CODE REVIEW

File: examples/Mediawiki/TemplateTransclusion.yaml [hunk 1]
```diff
@@ -28,32 +28,8 @@ tests:
...
-        response:
-          body: !pcre/pattern: /Testing template transclusion/
-      # Edit a template
-      - request:
-          method: post
-          path: api.php
-          parameters:
-            action: edit
+            text: "{{APIParsing}}"
             title: Template:APIParsing
```
Comment (brightbyte) [inline examples/Mediawiki/TemplateTransclusion.yaml]: title should be some other page's title, like Foo

File: examples/Mediawiki/TemplateTransclusion.yaml [hunk 2]
```diff
@@ -28,32 +28,8 @@ tests:
...
-            format: json
-        response:
-          body: !pcre/pattern: /Testing template transclusion/
-      # Edit a template
-      - request:
-          method: post
-          path: api.php
-          parameters:
-            action: edit
+            text: "{{APIParsing}}"
```
Comment (brightbyte) [inline examples/Mediawiki/TemplateTransclusion.yaml]: Maybe have some text before and after

File: examples/Mediawiki/TemplateTransclusion.yaml (hunk 1 above)
Comment (clarakosi) [reply examples/Mediawiki/TemplateTransclusion.yaml]: Done.

File: examples/Mediawiki/TemplateTransclusion.yaml (hunk 2 above)
Comment (clarakosi) [reply examples/Mediawiki/TemplateTransclusion.yaml]: Done.

**Pull request merged**